lib/matplotlib/axes/_axes.py

```diff
@@ -2469,105 +2469,105 @@ def stem(self, *args, **kwargs):
-          The radius of the pie, if *radius* is *None* it will be set to 1.
+        radius : float, optional (default is None)
+            The radius of the pie, if *radius* is *None* it will be set to 1.
 
-          *counterclock*: [ *False* | *True* ]
+        counterclock : bool, optional (default is True)
             Specify fractions direction, clockwise or counterclockwise.
 
-          *wedgeprops*: [ *None* | dict of key value pairs ]
+        wedgeprops : dict, optional (default is None)
             Dict of arguments passed to the wedge objects making the pie.
-            For example, you can pass in wedgeprops = { 'linewidth' : 3 }
+            For example, you can pass in``wedgeprops = {'linewidth': 3}``
             to set the width of the wedge border lines equal to 3.
             For more details, look at the doc/arguments of the wedge object.
-            By default `clip_on=False`.
+            By default ``clip_on=False``.
 
-          *textprops*: [ *None* | dict of key value pairs ]
+        textprops : dict, optional (default is None)
             Dict of arguments to pass to the text objects.
 
-          *center*: [ (0,0) | sequence of 2 scalars ]
-          Center position of the chart.
+        center :  list of float, optional (default is (0, 0))
+            Center position of the chart. Takes value (0, 0) or is a
+            sequence of 2 scalars.
 
-          *frame*: [ *False* | *True* ]
-            Plot axes frame with the chart.
+        frame : bool, optional (default is False)
+            Plot axes frame with the chart if true.
 
-          *rotatelabels*: [ *False* | *True* ]
-            Rotate each label to the angle of the corresponding slice.
+        rotatelabels : bool, optional (default is False)
+            Rotate each label to the angle of the corresponding slice if true.
 
-        The pie chart will probably look best if the figure and axes are
-        square, or the Axes aspect is equal.  e.g.::
-
-          figure(figsize=(8,8))
-          ax = axes([0.1, 0.1, 0.8, 0.8])
+        Returns
+        -------
+        patches : list
+            A sequence of :class:`matplotlib.patches.Wedge` instances
 
-        or::
+        texts : list
+            A is a list of the label :class:`matplotlib.text.Text` instances.
 
-          axes(aspect=1)
+        autotexts : list
+            A is a list of :class:`~matplotlib.text.Text` instances for the
+            numeric labels. Is returned only if parameter *autopct* is
+            not *None*. 
 
-        Return value:
-          If *autopct* is *None*, return the tuple (*patches*, *texts*):
+        Examples
+        --------
+        The pie chart will probably look best if the figure and axes are
+        square, or the Axes aspect is equal.
 
-            - *patches* is a sequence of
-              :class:`matplotlib.patches.Wedge` instances
+        .. plot:: mpl_examples/pie_and_polar_charts/pie_demo_features.py
```

Comment:
On matplotlib, I am not sure, as our pyplot API page is gigantic, but on sklearn it looks like this:
http://scikit-learn.org/stable/modules/generated/sklearn.isotonic.IsotonicRegression.html#sklearn.isotonic.IsotonicRegression

If you scroll down, you'll see all examples using the isotonic regression (there's only one).
I am not sure how to activate this feature just yet, but that is my next step :)